src/commands/package.ts

```diff
@@ -36,21 +39,37 @@ export default class Package extends Command {
             return;
         }
 
-        const compiler = new AppCompiler(this, fd);
-        const report = compiler.logDiagnostics();
+        const compiler = new AppsCompiler({
+            tool: packageInfo.name,
+            version: packageInfo.version,
+            when: new Date(),
+        }, this.getTypescriptForApp(fd) as any); // typing this is hard
```

Comment:
shouldn't this be abstracted out and called by both package and submit commands?